mixins/imageHelper.ts

```diff
@@ -100,16 +276,32 @@ const imageHelper = Vue.extend({
         maxWidth = element?.clientWidth,
         maxHeight = element?.clientHeight,
         quality = 90,
-        limitByWidth = false
+        limitByWidth = false,
+        backdropIndex = 0,
+        checkParent = true
       }: ImageUrlForElementParams
-    ): string {
+    ): string | undefined {
       if (item) {
-        if (!item.ImageTags) {
-          throw new TypeError(
-            'item.ImageTags must not be null or undefined when an item object is passed'
-          );
+        if (!tag) {
+          tag = this.getImageTag(item, type, backdropIndex, false);
+          if (!tag && checkParent) {
+            tag = this.getImageTag(item, type, backdropIndex, true);
+            itemId = this.getParentId(item);
+            if (!tag || !itemId) {
+              // eslint-disable-next-line no-console
+              console.error(
```

Comment:
It's probably not needed. If nothing has images, it'll just spam the console with errors.